src/iGameGUI.c

```diff
@@ -449,19 +455,39 @@ struct ObjApp * CreateApp(void)
 		MUIA_Menuitem_Title, GetMBString(MSG_MNMainProperties),
 		MUIA_Menuitem_Shortcut, MENU_PROPERTIES_HOTKEY,
 		End;
-
+	
+	MNMainOpenCurrentDir = MenuitemObject,
+		MUIA_Menuitem_Title, GetMBString(MSG_MNMainOpenCurrentDir),
```

Comment:
When we add new menu items or new features in the app, it would be nice to update also the guide file, explaining to the users what this is and how they can use it. This way we will keep the guide updated as much as possible.

Reply:
I've pushed up a change that added a one-liner about the feature in the guide.